src/mappings/storage/utils.ts

```diff
@@ -206,3 +214,134 @@ export async function deleteDataObjectsByIds(
   subtitlesRepository.remove(...currentSubtitles.flat())
   await deleteDataObjects(overlay, block, indexInBlock, extrinsicHash, objects)
 }
+
+export async function processSetNodeOperationalStatusMessage(
+  overlay: EntityManagerOverlay,
+  block: Block,
+  indexInBlock: number,
+  extrinsicHash: string | undefined,
+  workingGroup: 'storageWorkingGroup' | 'distributionWorkingGroup',
+  actor: Worker | undefined, // worker who sent the message, undefined if it's a lead
+  meta: ISetNodeOperationalStatus
+): Promise<void> {
+  let workerId: bigint
+  const bucketId = meta.bucketId || ''
+
+  // Get the bucket's worker id
+  if (actor) {
+    workerId = actor.runtimeId
+  } else {
+    const maybeWorker = await overlay
+      .getRepository(Worker)
+      .getById(`${workingGroup}-${meta.workerId}`)
+
+    if (!maybeWorker) {
+      return invalidMetadata(
```

Comment:
We return `invalidMetadata` here but `processNodeOperationalStatusMetadata` in `mappings/storage/metadata.ts` only calls it, should we be consistent about it?

Reply:
I think the return is just to break the function, alternatively we could just do 

```ts
invalidMetadata()
return 
```